lib/screens/explore_screens/regions_screens/regions_main/components/region_event_card/region_event_card.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:intl/intl.dart';
+import 'package:seeds/components/profile_avatar.dart';
+import 'package:seeds/datasource/remote/model/firebase_models/region_event_model.dart';
+import 'package:seeds/datasource/remote/model/profile_model.dart';
+import 'package:seeds/design/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/screens/explore_screens/regions_screens/regions_main/components/region_event_card/interactor/viewmodels/region_event_card_bloc.dart';
+import 'package:seeds/utils/build_context_extension.dart';
+
+class RegionEventCard extends StatelessWidget {
+  final RegionEventModel regionEvent;
+
+  const RegionEventCard(this.regionEvent, {Key? key}) : super(key: key);
+
+  Widget _buildMemberImages(List<ProfileModel> profiles) {
```

Comment:
Don’t split your widgets into methods
What happens internally is that when we make a change in and refresh the entire widget, it will also refresh the widgets that we have within the method, which leads us to waste CPU cycles.

What we should do is convert those methods to stateless Widget

Reply:
in this case its not so bad, since you only build it once. and not a list of them. but we should we moving this into a Widget in the component folder

Reply:
Moved to a widget